Dockerfile

```diff
@@ -26,10 +26,12 @@ RUN  \
 
 FROM alpine:3.17.0 AS server
 
-RUN apk update && apk add libtbb=2021.7.0-r0
+RUN apk update && apk add libtbb=2021.7.0-r0 curl jq
 
 WORKDIR /app
 COPY testBaseData .
 COPY --from=builder /src/siloApi .
 
+HEALTHCHECK CMD curl --fail --silent localhost:8080/info | jq .sequenceCount | xargs test 0 -ne || exit 1
```

Comment:
My shell skills are quite rusty. Could you please explain what it does and maybe add a short comment to the file?

Reply:
Done. Is it clear enough?